docs/js.md

```diff
@@ -206,26 +206,26 @@ Returns **[Promise][21]\<ModelResult>** A dictionary containing the top label an
 
 [18]: #parameters-7
 
-[19]: https://github.com/google/magika/blob/48d4394825d19221b6413203469a549b15a841ae/js/magika_node.ts#L29-L134 "Source code on GitHub"
+[19]: https://github.com/MichaelHinrichs/magika/blob/ce2e897198693c9c07c5e0ae72ea2a5a4d03d96e/js/magika_node.ts#L29-L134 "Source code on GitHub"
```

Comment:
@MichaelHinrichs This should be changed back to `github.com/google`

Reply:
I don't know how it got like that. I never touched that file.

Reply:
Should i change the folder between "blob" and "js" back too?